libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestClient.java

```diff
@@ -351,8 +369,10 @@ public void onFailure(Call call, IOException e) {
 
 			@Override
 			public void onResponse(Call call, Response response) throws IOException {
+            	// TODO: Remove onSuccess in 9.0.
 			    callback.onSuccess(restRequest, new RestResponse(response));
```

Comment:
If someone implements both, could that be an issue?
The response should not be consumed twice.